### PR TITLE
[ineco] initial plugin

### PR DESCRIPTION
**Pull request by karpoff**
plugin for Armenian bank Ineco

### CODE REVIEW

**Comment by karpoff** [comment]
@skvav are you the right person for review?

File: src/common/session.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
+import { stringify } from 'querystring'
+import * as setCookie from 'set-cookie-parser'
+import { fetch } from './network'
+
+/**
+ * Session is used for auto storing cookies within all requests
+ * It also provide simple interfaces for major request types
+ */
+export class Session {
```
Comment (skvav) [inline src/common/session.js:9]: There is no need to provide explicit cookie handling. They are managed implicitly. If it doesn't work with your bank please place session specific code in plugin directory.

File: src/common/utils.js [hunk 2]
```diff
@@ -70,3 +70,39 @@ export async function takePicture (format) {
     })
   })
 }
+
+export function parseCSV (str) {
```
Comment (skvav) [inline src/common/utils.js:74]: Please move it to your plugin code. CSV is quite flexible format and doesn't specify standard separator. Widely used parsing function must take it into consideration or be just a helper for its plugin.

File: src/plugins/ineco/index.js [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+          tr.opIncomeInstrument = outcomeCurrency
+        }
+      }
+    }
+
+    trans.push(tr)
+  }
+
+  return trans
+}
```
Comment (skvav) [inline src/plugins/ineco/index.js:117]: It would be great if you place your converting code into converters.js and provide tests for common cases :)

**Comment by skvav** [comment]
@karpoff Sorry for late response

File: src/common/session.js (hunk 1 above)
Comment (karpoff) [reply src/common/session.js:9]: What do you mean that they are managed implicitly? I want to automatically store set cookies within requests, 'fetch' doesn't have such behavior
I've moved this class to plugin directory

File: src/common/utils.js (hunk 2 above)
Comment (karpoff) [reply src/common/utils.js:74]: done

File: src/plugins/ineco/index.js (hunk 3 above)
Comment (karpoff) [reply src/plugins/ineco/index.js:117]: moved
will add tests. is it blocks going to prod?)

Comment (karpoff) [reply src/plugins/ineco/index.js]: tests are added

**Comment by karpoff** [comment]
@skvav all comments are resolved

**Comment by skvav** [comment]
@karpoff finally it got to prod :)

**Pull request merged**